packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -363,6 +408,57 @@ class LengthLimitingTextInputFormatter extends TextInputFormatter {
   /// composing is not allowed.
   final int? maxLength;
 
+  /// Determines how the [maxLength] limit should be enforced.
+  ///
+  /// Defaults to [MaxLengthEnforcement.enforced].
+  ///
+  /// {@macro flutter.services.textFormatter.maxLengthEnforcement}
+  final MaxLengthEnforcement? maxLengthEnforcement;
+
+  /// Return an effective [MaxLengthEnforcement] according the target platform.
+  ///
+  /// if [previousEnforcement] is not null, return it directly.
+  ///
+  /// if non of [platform] was specified, [defaultTargetPlatform] will be use.
```

Comment:
"If platform was not specified, [defaultTargetPlatform] will be used."

I think [platform] doesn't link to anything, does it?

Reply:
[platform] here links an argument.